articles/iot-edge/how-to-deploy-cli-at-scale.md

```diff
@@ -162,6 +162,23 @@ The previous example showed a layered deployment setting the `properties.desired
 }
 ```
 
+The same can also be expressed with:
+
+```json
+"SimulatedTEmperatureSensor": {
+  "properties.desired.layeredProperties.SendData" : true,
+  "properties.desired.layeredProperties.SendInterval": 5
+}
+```
+
+> Warning: Currently there is a bug with a layered deployment that updates only properties. These layers need an empty edgeAgent object (i.e. `"$edgeAgent":{}`) in the deployment manifest to be considered a valid layered deployment. They will also only be shown as applied in the module twins they target. For the `$edgeAgent` module twin, and hence in the Azure portal, the layer will only ever be marked as targeted, not applied.
```

Comment:
A suggestion:

```suggestion
>[!NOTE]
>Currently, all layered deployments must include an edgeAgent object to be considered valid. Even if a layered deployment only updates module properties, include an empty object. For example, `"$edgeAgent":{}`. A layered deployment with an empty edgeAgent object will be shown as **targeted** in the edgeAgent module twin, not **applied**.
```

The [!NOTE] syntax will give you the nice purple box bounding this text.